lib/pure/math.nim

```diff
@@ -65,6 +65,8 @@ when defined(c) or defined(cpp):
   proc c_copysign(x, y: cfloat): cfloat {.importc: "copysignf", header: "<math.h>".}
   proc c_copysign(x, y: cdouble): cdouble {.importc: "copysign", header: "<math.h>".}
 
+  proc c_signbit(x: SomeFloat): cint {.importc: "signbit", header: "<math.h>".}
```

Comment:
Is signbit always implemented as a macro in c? Otherwise we may need non generics here

Reply:
```
7.12.3.6 The signbit macro
Synopsis
1 #include <math.h>
int signbit(real-floating x);
Description
2 The signbit macro determines whether the sign of its argument value is negative.198)
Returns
3 The signbit macro returns a nonzero value if and only if the sign of its argument value
is negative.
```

Reply:
and C++11 also
C11 standard (ISO/IEC 9899:2011):
https://en.cppreference.com/w/c/numeric/math/signbit

ok